homeassistant/components/venstar/climate.py

```diff
@@ -318,13 +311,9 @@ def _set_temperature(self, **kwargs):
 
             if not success:
                 _LOGGER.error("Failed to change the temperature")
+        self.schedule_update_ha_state()
```

Comment:
@bdraco how much does this cost? I think the simpler code in this version is nicer to read and maintain.

Reply:
It is quite small especially considering how infrequently we expect this to be called.  

Either way works for this case unless we expect we will need to do more in async context later.

Reply:
I will leave this as-is unless you say otherwise?